File: L9/Java/14.js
let palavra = prompt("Digite uma palavra:");
let vogais = ['a', 'e', 'i', 'o', 'u'];
let numVogais = 0;

for (let i = 0; i < palavra.length; i++) {
  if (vogais.includes(palavra[i].toLowerCase())) {
    numVogais++;
  }
}

let caractere = prompt("Digite um caractere:");

for (let i = 0; i < palavra.length; i++) {
  if (vogais.includes(palavra[i].toLowerCase())) {
    palavra = palavra.substring(0, i) + caractere + palavra.substring(i + 1);
  }
}

console.log(`Número de vogais: ${numVogais}`);
console.log(`Palavra com vogais substituídas: ${palavra}`);

File: L9/Python/14.py
palavra = input("Digite uma palavra: ")

vogais = ['a', 'e', 'i', 'o', 'u']
quantidade_vogais = 0

for letra in palavra:
    if letra.lower() in vogais:
        quantidade_vogais += 1

caractere_substituto = input("Digite um caractere: ")

nova_palavra = palavra.replace('a', caractere_substituto)\
                     .replace('e', caractere_substituto)\
                     .replace('i', caractere_substituto)\
                     .replace('o', caractere_substituto)\
                     .replace('u', caractere_substituto)

print(f"A palavra '{palavra}' possui {quantidade_vogais} vogais.")
print(f"A nova palavra é '{nova_palavra}'.")
